### DISCUSSION TITLE
Error on kb layout being displayed

### DISCUSSION THREAD

**Discussion by Alanzcc** [end-4/dots-hyprland#380] [Q&A]
I set 2 keyboard layouts on hyprland br and us. When I press Super + Space to change it displays the current layout on top right, but it doesn't change again after the first change, so if I started with br and changed to us it would be stuck displaying us on the bar even if I went back to br. Anyone knows a fix?

**Comment by end-4** [comment]
were you using many apps? i think the kb layout is remembered per window.

**Reply by Alanzcc** [reply]
No, after it displays the first change in kb layout with or without many apps open when that happens, it just won't update the display on the top right.

**Comment by JxJxxJxJ** [comment]
Were you able to solve it? Just ran into the same issue

**Reply by AhmadRehan98** [reply]
were you? I also have the same issue :D

**Reply by Alanzcc** [reply]
When I still used these dots I never found a way to fix it.
